Src/Plugins/Tests.Security/Validators/SEC101_007.GitHubAppCredentialsValidatorTests.cs

```diff
@@ -84,7 +84,7 @@ public void GitHubAppCredentialsValidator_MockHttpTests()
                 var fingerprint = new Fingerprint(fingerprintText);
                 var keyValuePairs = new Dictionary<string, string>();
 
-                using var httpClient = new HttpClient(HttpMockHelper.Mock(testCase.HttpStatusCode, testCase.HttpContent));
+                using var httpClient = new HttpClient(Helpers.HttpMockHelper.Mock(testCase.HttpStatusCode, testCase.HttpContent));
```

Comment:
>Helpers. [](http://example.com/codeflow?start=54&length=8)

the sarif code now contains the same httpmock infrastructure.

will have to check if we should drop from this or something else.